app/client/src/config/fields.ts

```diff
@@ -271,7 +271,8 @@ export const filterFields = [
   },
   {
     key: 'actionId',
-    label: 'Action ID',
+    label: 'Action ID (Action Name)',
+    placeholder: 'Select an Action',
```

Comment:
```suggestion
    placeholder: 'Select an Action...',
```

I think these should have the `...` to be consistent with the other drop downs or the `...` should be removed from the other drop downs.

Reply:
Good catch!